modules/buy-button/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@vtex/buy-button",
+  "version": "0.0.6",
+  "description": "Component used to call a mutation that adds one item to the cart",
+  "main": "build/index.js",
+  "peerDependencies": {
+    "react": "^15.5.4"
+  },
+  "files": [
+    "lib/"
+  ],
+  "dependencies": {
+    "@vtex/styleguide": "2.0.0-rc.34",
+    "apollo-client": "^2.2.3",
```

Comment:
always use ~, ^ will apply any 2.x versions, this may break the code. ~ will apply any 2.2.x versions, which is a fix version accordingly with semver.

Reply:
👍